server/models/action-inputs-model.js

```diff
@@ -4,18 +4,30 @@ function getTitle (actionId) {
 
 function getHint (actionId, parcelRef) {
   // TODO: Unsafe use of potentially unvalidated user input.
-  return actionId === 'FG1' ? `How long is the fence you want to claim on in metres for ${parcelRef}?` : `What is the area you wish to claim on in hectares for ${parcelRef}?`
+  return actionId === 'FG1' ? 'How long is the fence you want to claim on in metres for ?' : 'What is the area you wish to claim on in hectares for ?'
 }
 
-function actionsInputsModel (parcelRef, actionId, error, parcels) {
+function boundsValid (action) {
+  return action && action.inputs && action.inputs[0] && action.inputs[0].upperbound && action.inputs[0].lowerbound
+}
+
+function boundsMessage (action) {
+  return `Please choose a value between ${action.inputs[0].lowerbound} and ${action.inputs[0].upperbound}`
+}
+
+function getBoundsHint (action) {
+  return boundsValid(action) ? boundsMessage(action) : ''
+}
+
```

Comment:
The rebase seems to have lost the improvements to the get bounds code from the commit https://github.com/DEFRA/ffc-ce-web/pull/26/commits/2d71feb209f6699ee0f46a64dfe113067613e07e

the previous implementation for getting the hint obtained the input field in a more robust fashion via a find, rather than getting the first item in the list. Passing back the item with the input also removed multiple calls to inputs[0] in the code.
```
function getBounds (action) {
  return boundsValid(action)
    ? action.inputs.find(input => input.upperbound !== undefined && input.lowerbound !== undefined)
    : undefined
}

function boundsMessage (bounds) {
  return `Please choose a value between ${bounds.lowerbound} and ${bounds.upperbound}.`
}

function getBoundsHint (action) {	
  const bounds = getBounds(action)
  return bounds ? boundsMessage(bounds) : ''
}

```

Reply:
Yep, doesn't surprise me that  missed something.

If you want to move this PR on today then feel free to rebase #19 or make the changes here.